frontend/apps/reader/modules/readerlink.lua

```diff
@@ -61,6 +61,10 @@ local function swipe_to_go_back()
     return G_reader_settings:readSetting("swipe_to_go_back") == true
 end
 
+local function swipe_to_follow_first_link()
```

Comment:
NB `is_follow_links_on()` and `swipe_to_go_back()`. It'd be cool if you could update those as well.